core/storage/exploration/gae_models.py

```diff
@@ -169,7 +169,9 @@ class ExplorationRightsModel(base_models.VersionedModel):
     owner_ids = ndb.StringProperty(indexed=True, repeated=True)
     # The user_ids of users who are allowed to edit this exploration.
     editor_ids = ndb.StringProperty(indexed=True, repeated=True)
-    # The user_ids of users who are allowed to translate this exploration.
+    # The user_ids of users who are allowed to voiceover this exploration.
+    voice_artist_ids = ndb.StringProperty(indexed=True, repeated=True)
+    # DEPRECATED in v2.8.1. Do not use.
     translator_ids = ndb.StringProperty(indexed=True, repeated=True)
```

Comment:
Move this value at the end (like we have done in other places [line 89])

Reply:
Done.